distribution/changelog.txt

```diff
@@ -22,6 +22,7 @@
 - Fix: [#22222] Staff list may remain invalid when changing tabs.
 - Fix: [#22265] Button for switching covered tracks on slides doesn’t stay pressed.
 - Fix: [#22284] Unrated rides cause high amount of nausea.
+- Fix: [#22292] Progress bar not updating correctly with multiple windows open.
```

Comment:
Could you specify this is about _widgets_ in guest and ride windows?

Reply:
Will do